# Because there is no configure script, one has to specify the compiler manually.
# By Toon Verstraelen <toon.verstraelen@ugent.be>
--- molden5.0-orig/makefile	2013-11-28 16:02:04.563545000 +0100
+++ molden5.0/makefile	2013-11-28 16:08:13.606238000 +0100
@@ -13,8 +13,8 @@
 #	Disregard compiler warnings on rdmolf.f
 #
 #
-CC = cc
-FC = gfortran
+CC = icc
+FC = ifort
 LIBS =  -lX11 -lm
 LDR = ${FC} 
 LIBSG = -L/usr/X11R6/lib -lGLU -lGL -lXmu -lX11 -lm
